package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-day-picker",
-  "version": "7.3.0",
+  "version": "7.4.0",
```

Comment:
Maybe we use pre-release semver tags to clarify that we are forking 

`7.4.0-lola` ?